server/sonar-web/src/main/js/apps/organizations/store/reducer.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import uniq from 'lodash/uniq';
+import { actions } from './membersDuck';
+
+const organizationMembers = (state = {}, action = {}) => {
+  const organization = state[action.organization] || {};
+  switch (action.type) {
+    case actions.UPDATE_STATE:
+      return {
+        ...state,
+        [action.organization]: {
+          ...organization,
+          ...action.changes
+        }
+      };
+    case actions.RECEIVE_MEMBERS:
+      return {
+        ...state,
+        [action.organization]: {
+          ...organization,
+          members: action.members.map(member => member.login)
+        }
+      };
+    case actions.RECEIVE_MORE_MEMBERS: {
+      return {
+        ...state,
+        [action.organization]: {
+          ...organization,
+          members: uniq(
+            getOrganizationMembersLogins(state, action.organization).concat(
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Move the declaration of "getOrganizationMembersLogins" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)